src/lib/content/coordinators/GetContentItem.spec.ts

```diff
@@ -202,6 +218,29 @@ describe('GetContentItem', () => {
         ]
       });
     });
+
+    it('should inline content references', () => {
```

Comment:
I don't think inline is the right word. We inline content-links, meaning that we replace the link with the actual content. For content references, we just parse it into a model but leave it as a reference and not fetch the content.

Reply:
would should attach reference be more appropriate?

Reply:
👍